### DISCUSSION TITLE
Contact point.

### DISCUSSION THREAD

**Discussion by PatrickEasy** [DeFiCh/app#741] [Q&A] [answered]
Hi, 

I am reaching out on behalf of a customer of EasyCrypto.ai. 

They have had technical difficulties in regards to what we believe is [this issue.](https://github.com/DeFiCh/app/issues/475)

Are we able to get in contact with someone who can provide assistance, that either we can relay to the customer or that can put in contact with them directly? 

The primary contact is help@easycrypto.ai and lead with Attn: Patrick and I'll pass along my direct contact point. 

Any advice or assistance that can be provided, is greatly appreciated.

Cheers

Patrick
Easy Crypto Customer Support

**Comment by izzycsy** [comment, marked as answer]
Hi @PatrickEasy , the issue has been resolved and if it's with regards to syncing issues, the user can refer to DeFiChain's Telegram or the [official release notes](https://github.com/DeFiCh/app/releases/tag/v2.3.3). Hope that answers your enquiry.
